Lyric

Geertseb opened the pull request:
## Description:


**Related issue (if applicable):** fixes #<home-assistant issue number goes here>

**Pull request in [home-assistant.github.io](https://github.com/home-assistant/home-assistant.github.io) with documentation (if applicable):** home-assistant/home-assistant.github.io#<home-assistant.github.io PR number goes here>

## Example entry for `configuration.yaml` (if applicable):
```yaml

```

## Checklist:

If user exposed functionality or configuration variables are added/changed:
  - [ ] Documentation added/updated in [home-assistant.github.io](https://github.com/home-assistant/home-assistant.github.io)

If the code communicates with devices, web services, or third-party tools:
  - [ ] Local tests with `tox` run successfully. **Your PR cannot be merged unless tests pass**
  - [ ] New dependencies have been added to the `REQUIREMENTS` variable ([example][ex-requir]).
  - [ ] New dependencies are only imported inside functions that use them ([example][ex-import]).
  - [ ] New dependencies have been added to `requirements_all.txt` by running `script/gen_requirements_all.py`.
  - [ ] New files were added to `.coveragerc`.

If the code does not interact with devices:
  - [ ] Local tests with `tox` run successfully. **Your PR cannot be merged unless tests pass**
  - [ ] Tests have been added to verify that the new code works.

[ex-requir]: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/keyboard.py#L14
[ex-import]: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/keyboard.py#L54

bramkragten committed:
Lyric component

bramkragten committed:
Update lyric.py

bramkragten committed:
Bump

bramkragten committed:
Update lyric.py

bramkragten committed:
Update lyric.py

bramkragten committed:
Update lyric.py

bramkragten commented:
Why would you create this pull request? You might want to ask me to do it, as it is my code and we are testing it at the forum?

Geertseb commented:
Bram,Sorry, I want to setup the Lyric in HASS and tried to get the file but I don't know how to do this.I've copied the the content to my py but get errors about the 'Sensor'So I'm a bit lost how to set it up.On September 14, 2017 at 5:18 PM Bram Kragten <notifications@github.com> wrote:Why would you create this pull request? You might want to ask me to do it, as it is my code and we are testing it at the forum?—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread. 

bramkragten commented:
You should ask in the community like everyone else. https://community.home-assistant.io/t/honeywell-lyric-thermostat/3520

Geertseb commented:
Again sorry, I look there and tried to understand how ik works and I get lost. So yes i will ask there. I'm nog familiar with how te get those filesOn September 14, 2017 at 5:34 PM Bram Kragten <notifications@github.com> wrote:You should ask in the community like everyone else. https://community.home-assistant.io/t/honeywell-lyric-thermostat/3520—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread. 

arsaboo commented:
@Geertseb you can close this PR and let @bramkragten open a PR to add the Lyric component. He is familiar with the code and should be able to get it merged faster. In the meantime, you can use it as a custom component.

bramkragten committed:
Use hass.components to get configurator component

https://github.com/home-assistant/home-assistant/commit/19d1d748d447e425031d3b5da74e2121b472ceba

balloob commented:
Closing this as it's code by @bramkragten
